src/hotspot/share/gc/g1/g1Policy.cpp

```diff
@@ -513,6 +513,27 @@ double G1Policy::predict_survivor_regions_evac_time() const {
   return survivor_regions_evac_time;
 }
 
+double G1Policy::predict_retained_regions_evac_time() const {
```

Comment:
Seems that the result is only for log-print in its sole caller, `predict_base_time_ms`. However, it's also odd that this method is called there at all.

Reply:
This is a bug. It's supposed to be used for calculating the total_time in that method too. Fixed.